src/lib/engine/space.js

```diff
@@ -73,10 +73,19 @@ export function toDSpace(spaceId, graph, organizationFacts) {
 
 function toDgraph(space, graph){
   const {users, organizations, calculators, userOrganizationMemberships, me} = graph
-  const {user_id, organization_id} = space
+  const {user_id, organization_id, author_contributions} = space
+
+  let org_users = []
+
+  if (!!organization_id && !_.isEmpty(author_contributions)){
+    org_users = Object.keys(author_contributions).map(key => ({..._collections.get(users, key), edits: author_contributions[key]}))
```

Comment:
I would rename `key` to `author_id` or something here, I think that would be clearer.